sdcm/nemesis.py

```diff
@@ -3446,6 +3448,9 @@ def disrupt_network_random_interruptions(self):  # pylint: disable=invalid-name
         if not self.target_node.install_traffic_control():
             raise UnsupportedNemesis("Traffic control package not installed on system")
 
+        if not self.target_node.scylla_network_configuration.device:
+            raise UnsupportedNemesis("The network device name is not recognized")
```

Comment:
I'm not sure we want to silently skip, if this wasn't working as expected.

we want to know about it, so we'll be able to fix.

Reply:
If this exception will be raised, we will see the reason - why it was skipped. Or raise exception will be better?

Reply:
it would be written, but no one would looks at it.

raise an exception would serve us better here.

Reply:
raised `ValueError`